src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="form-field"> <label for="Surname">Surname:</label>
+          <input type="text" name="Surname"
+            id="Surname"
+            autocomplete="off">
+        </div>
+
+        <div class="form-field"> <label for="Name:">Name:</label>
+          <input
+            type="text"
+            name="Name"
+            id="Name"
+            autocomplete="off">
```

Comment:
```suggestion
            autocomplete="off"
          >
        </label>
```
you need to cover whole input in `label` tag

Reply:
Добрый день, Андрей. Спасибо за комментарий.
Я так поняла, что в эту школу, на этот курс приходят люди, которые уже что-то понимают в разработке, которым знаком интерфейс GitHub, которые четко понимают, что означает каждая ошибка npm... Но я начала 3 месяца назад с 0.

Reply:
И твоя рекомендация "you need to cover whole input in label tag" - она для чего, почему нельзя оставить так как у меня, в чем принципиальная ошибка?
Просто без объяснения, почему то, что я делаю - это не правильно, я никогда не научусь и не пойму суть

Reply:
Привет,  Олеся. Прошу на ты.
Если позволите я немного расскажу о своей истории:
Я пришел в Мейт с полного нуля в HTML и СSS. Все с чем я работал до Мейта это exel таблицы. Мне самому давался старт не очень легко но за счет программы которая выстроена как воронка, чем дальше по темам ты проходишь тем проще в них разбираться! 
Интенсивность обучения тут правда высокая, но менторска группа будет стараться вам помочь со болей частью возникших вопросов, в особенности с теми что касается ошибок/багов. и по этому если вопрос не до конца был решен дублируй его в QnA чате slack или тегай с помощь `@` менторов если сообщение утонуло.